### PR TITLE
Elementary support + readme update

### PR DESCRIPTION
**Pull request by bh90210**
ATM all supported linux distros are (from README):
#### Debian/Ubuntu

`sudo apt install libgtk-3-dev libwebkit2gtk-4.0-dev`

_Debian: 8, 9, 10_
_Ubuntu: 16.04, 18.04, 19.04_

_Also succesfully tested on: Zorin 15, Parrot 4.7, Linuxmint 19, Elementary 5_

#### Arch Linux

`sudo pacman -S webkit2gtk gtk3`

_Also succesfully test on: ArcoLinuxB_

#### Centos

`sudo yum install webkitgtk3-devel gtk3-devel`

_CentOS 6, 7_

#### Fedora

`sudo yum install webkit2gtk3-devel gtk3-devel`

_Fedora 29, 30_

#### VoidLinux & VoidLinux-musl

`xbps-install gtk+3-devel webkit2gtk-devel`

#### Gentoo

`sudo emerge gtk+:3 webkit-gtk`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,24 +46,37 @@ Make sure you have the xcode command line tools installed. This can be done by r
...
+#### Fedora
 
 `sudo yum install webkit2gtk3-devel gtk3-devel`
 
+_Fedora 29, 30_
+
+#### VoidLinux & VoidLinux-musl
+
+`xbps-install gtk+3-devel webkit2gtk-devel`
+
```
Comment (bh90210) [inline README.md]: updated the readme to reflect current status of all distros

File: cmd/cmd-mewn.go [hunk 2]
```diff
@@ -7,5 +7,5 @@ import "github.com/leaanthony/mewn"
...
...fdb21c10da792a25bc083f8727173203b4f084a36fa6d2206446d039d45e50f9045467e0ed0228a7426f202c724e98d133c14fe28d30fb2975a1fa72b1602510770b73ef0b974c2619a6636d32742658866366d4c4a10fc5a73fa6e357f80acbc5a5d4cd81c068041bd74a1650427c76db962a456aa915d8d9b86662435552a4dce7e49a64f87c522bdb40fb9b2945fa82692efcb44cfa73fcfb10fc5dc4ee42421ab40fd17e55dccf7e5b9d56d5f418a1f6d86f16a45cb3606b60830d2f77dbb0b5ef975bf7bda0d61a1febaee241eabe5f42c1cdff8f935ba1c3ebae0e5cae1f40672af68bb1a275c42ec3414afdd820bd6b7db26cb312bab59fcdd020752ef711fe5bc1bd6bb12851a1f6d42e62b5716c9072231cdc3df4a9f8fc7e7d36228bf596d120957e30223bdda4a7bd005ed3c291e6e6a70ecb931acf72e9af519de9b25fe3e87b436feaa0cfafaeab561c45ddcadf5f5febaa3c9ef12d489796a1f6c6451d57f1204afe1bb5bf7b80f716f3f22db208aa19e68c117675058ae678b6776c9bb0c8798940ce24ef0e1f162849bd7bcd63b5fdc1691e38d002ddfe37107d5d3000e95ba02ef50c3363697442150fe2847f23d41e0fecbcb67ef9600b5f97ea6753f87430894f779153cbe651e1653488beffb26cfe639cf305658a6a200f65662f5197423c92a2ce1efe308f14ba50478bf8d8c6f9fcfa10d91e491301ba64bc7c849828d42a1e44aa3711ea7bcb3438a1392c4cb0e016cea3facd95fdc829475054538ef6c473f738865eea3c047aef233a394c9ba8d072ac33fb4a931c2acd9a2566bf61fa160000ffff0fbe5d9f2c160000")
```
Comment (bh90210) [inline cmd/cmd-mewn.go]: mewn `linuxdb.yaml`

File: cmd/linux.go [hunk 3]
```diff
@@ -37,6 +37,8 @@ const (
 	Linuxmint
 	// VoidLinux distribution
 	VoidLinux
+	// Elementary distribution
```
Comment (bh90210) [inline cmd/linux.go]: new entry for elementary distro

File: cmd/linuxdb.yaml [hunk 4]
```diff
@@ -55,6 +55,15 @@ distributions:
         gccversioncommand: *gccdumpversion
         programs: *debiandefaultprograms
         libraries: *debiandefaultlibraries
+  elementary:
```
Comment (bh90210) [inline cmd/linuxdb.yaml]: db entry

File: cmd/system.go [hunk 5]
```diff
@@ -274,7 +274,7 @@ func CheckDependencies(logger *Logger) (bool, error) {
 		distroInfo := GetLinuxDistroInfo()
 
 		switch distroInfo.Distribution {
-		case Ubuntu, Debian, Zorin, Parrot, Linuxmint:
+		case Ubuntu, Debian, Zorin, Parrot, Linuxmint, Elementary:
```
Comment (bh90210) [inline cmd/system.go]: check dependencies entry

**Comment by Vaelatern** [comment]
At the next release (tonight?) I plan on packaging wails for Void Linux so our users just `xbps-install wails`

**Comment by leaanthony** [comment]
@Vaelatern The next release will be in a few weeks and we'll feature freeze for V1. It would be awesome if v1 was packaged but it sounds like it's a rolling distro so no problems updating? Also, we want to make sure we nail that rendering issue for Void.

I don't suppose you have any thoughts on [this issue](https://github.com/wailsapp/wails/issues/2) do you?

**Pull request merged**